pyccel/codegen/printing/ccode.py

```diff
@@ -274,7 +274,7 @@ def _print_ModuleHeader(self, expr):
         funcs = '\n\n'.join('{};'.format(self.function_signature(f)) for f in expr.module.funcs)
 
         # Print imports last to be sure that all additional_imports have been collected
-        imports = [*expr.module.imports, *map(Import, self._additional_imports)]
+        imports = [*expr.module.imports,*map(Import, self._additional_imports)]
```

Comment:
Not sure why you changed this file

Reply:
this was revert changes.
if you follow the commits you will see that.

Reply:
Ah so you changed the file then changed it back and the only difference left is the removed space. In that case please checkout this file from the master branch so it is clear that it is unchanged